.github/CODEOWNERS

```diff
@@ -25,20 +25,20 @@
 /src/platforms/rust/ @getsentry/owners-native
 /src/wizard/rust/ @getsentry/owners-native
 
-/src/platforms/javascript/ @getsentry/team-webplatform
-/src/wizard/javascript/ @getsentry/team-webplatform
-/src/platforms/php/ @getsentry/team-webplatform
-/src/wizard/php/ @getsentry/team-webplatform
-/src/platforms/go/ @getsentry/team-webplatform
-/src/wizard/go/ @getsentry/team-webplatform
-/src/platforms/elixir/ @getsentry/team-webplatform
-/src/wizard/elixir/ @getsentry/team-webplatform
-/src/platforms/python/ @getsentry/team-webplatform
-/src/wizard/python/ @getsentry/team-webplatform
+/src/platforms/javascript/ @rhcarvalho
+/src/wizard/javascript/ @rhcarvalho
+/src/platforms/php/ @rhcarvalho
+/src/wizard/php/ @rhcarvalho
+/src/platforms/go/ @rhcarvalho
+/src/wizard/go/ @rhcarvalho
+/src/platforms/elixir/ @rhcarvalho
+/src/wizard/elixir/ @rhcarvalho
+/src/platforms/python/ @rhcarvalho
+/src/wizard/python/ @rhcarvalho
 
 /src/docs/product/discover-queries/ @getsentry/visibility
 /src/docs/product/performance/ @getsentry/visibility
 
 /src/docs/product/cli/ @kamilogorek
-/src/docs/product/cli/releases/ @getsentry/team-webplatform
+/src/docs/product/cli/releases/ @rhcarvalho
```

Comment:
You don't want to it? :P

Reply:
Hehe, from the contents alone, it is not clear how https://docs.sentry.io/product/cli/releases/ differs from https://docs.sentry.io/product/cli/ from a triaging pov

Reply:
@rhcarvalho I don't really know why that is, but since it's not related to my change (and PR approved) I'll merge but feel free to open a follow up with more improvements